packages/wallet/src/test-utils/defaultChainConfig.ts

```diff
@@ -0,0 +1,410 @@
+export const defaultChainConfig = {
+  chain_name: 'local_testnet',
+  block_gas_limit: 5000000000,
+  initial_state: {
+    coins: [
+      {
+        owner: '0x94ffcc53b892684acefaebc8a3d4a595e528a8cf664eeb3ef36f1020b0809d0d',
```

Comment:
This default chain config is a direct copy of the chain config file we use for our local dev environment. It is there just to provide a good default to users who don't want customization, and want something up and running quickly.

Reply:
I agree, but could you please create a new issue for this?

I'm also _[erroneously]_ duplicating this file in another PR I'm working on.

This will give us time to ponder the best way to resolve all occurrences simultaneously.